shared/AppInsightsCore/src/JavaScriptSDK/InstrumentHooks.ts

```diff
@@ -109,15 +110,17 @@ function _createFunctionHook(aiHook:IInstrumentHooks) {
 
         // Call the original function was called
         let theFunc = aiHook.f;
-        try {
-            funcArgs.rslt = theFunc.apply(funcThis, orgArgs);
-        } catch (err) {
-            // Report the request callback
-            funcArgs.err = err;
-            _doCallbacks(hooks, funcArgs, cbArgs, hookCtx, CallbackType.FunctionError);
-
-            // rethrow the original exception so anyone listening for it can catch the exception
-            throw err;
+        if (theFunc) {
```

Comment:
This was 1 issue in that there was no previous "event" to call, so `theFunc.apply(...)` would throw and call the error handling hook and not the processing hook